solutions/cses-1641.mdx

```diff
@@ -51,3 +56,77 @@ int main(){
 	cout << "IMPOSSIBLE" << endl;
 }
 ```
+	
+</CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class SumOfThreeValues {
+	static class comparator implements Comparator<Group> {
```

Comment:
can't you just put this as a part of `Group` or something

Reply:
yeah u can make group extend the Comparable class

Reply:
im not sure how to do that because I've only done this before in priority queues